SECURITY.md

```diff
@@ -0,0 +1,13 @@
+# Security Policy
+
+## Supported Versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| < 1.0.0 | :x:                |
+| 1.x.x   | :white_check_mark: |
+| 2.x.x   | :white_check_mark: |
```

Comment:
This should IMO leave on the docs site, close (maybe in a separate file, or not) to https://docs.konghq.com/kubernetes-ingress-controller/1.1.x/references/version-compatibility/#kubernetes

Reply:
:+1: 283e1a8a3298b923241eeae40a0a21855cdec0d8